website/src/content/docs/docs/howtos/Generate client libraries/04method.mdx

```diff
@@ -148,6 +161,10 @@ public final class PetStoreNamespaceClient {
 }
 ```
 
+```go
```

Comment:
Don't quite understand this sentense.
Does this mean 'The generated code in Go should remain unchanged between protocolAPI and convenientAPI.'?

Reply:
Actually Go does not support protocol method generation. I suggest to document not supported here.